aiqc/__init__.py

```diff
@@ -741,13 +741,10 @@ def to_pandas(
 			samples:list = None
 		):
 			df = Dataset.Tabular.to_pandas(id, columns, samples)
-
 			if Dataset.Text.column_name not in columns:
```

Comment:
When phasing out Dataset.Text for tabular, you will probably be able to keep methods like `from_folder`